lib/features/digital_guide_view/presentation/widgets/digital_guide_go_to_button.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+
+import "../../../../config/ui_config.dart";
+import "../../../../theme/app_theme.dart";
+import "../../tabs/entraces/data/models/digital_guide_entrace.dart";
+
+class DigitalGuideGoToButton extends StatelessWidget {
+  const DigitalGuideGoToButton({
+    super.key,
+    required this.onTap,
+    required this.entrace,
+  });
+
+  final VoidCallback onTap;
+  final DigitalGuideEntrace entrace;
```

Comment:
Also I suppose this will work for other models too in the future, not only entrances? So maybe some more general parameters could be applied here? And then some `EntranceNavLink` that uses the `DigitalGuideGoToButton` /`DigitalGuideNavLink` inside. If you know what I mean